app/HutchManager/Controllers/UsersController.cs

```diff
@@ -63,6 +63,10 @@ public async Task<IActionResult> SetUICulture([FromBody] string culture)
     return NoContent();
   }
 
+  [HttpGet]
+  public async Task<List<UserModel>> List()
```

Comment:
actually, can you move list and create to a new `UsersController` and leave this one called `UserController`.

The `UserController` that was already in the codebase is intended for interactions for **the current user** e.g. `/user/me` gets the current authenticated user's profile.

I believe that should be different than the rest api for interacting with user resources (which should be UsersController plural, and have things like `GET /users` to list and `POST /users` to create.